3.x/zh_CN/docs/operation_and_maintenance/console/console_commands.md

```diff
@@ -2567,7 +2567,7 @@ Load account 0x6fad87071f790c3234108f41b76bb99874a6d813 success!
 
 ### 3. listAccount
 
-查看当前加载的所有账户信息
+查看当前加载的所有账户信息。但是，若控制台使用密码机的公私钥，因公私钥都放在密码机内部，所以使用不了该操作命令。
```

Comment:
```suggestion
查看当前加载的所有账户信息。若控制台使用密码机的公私钥，因公私钥都放在密码机内部，所以使用不了该操作命令。
```

Reply:
has support listAccount command in HSM model